packages/asset-swapper/src/utils/market_operation_utils/index.ts

```diff
@@ -663,17 +666,49 @@ export class MarketOperationUtils {
             // Timing of RFQT lifecycle
             const timeStart = new Date().getTime();
             const { makerToken, takerToken } = nativeOrders[0].order;
```

Comment:
[out of scope] it seems like `fooToken` sometimes mean the symbol(?) (e.g. `RfqClientV1PriceRequest` has both 
`takerAddress`, and `takerToken`) while here it's address.